automount.sh

```diff
@@ -29,7 +29,7 @@ function mount_drive () {
   else
     #TODO: Better default options
     echo "Attempting Mounting $fs_type"
-    mount "$1" "/run/media/deck/$label"
+    mount "$1" "/run/media/deck/$label" -ouid=1000,gid=1000,user
```

Comment:
If the purpose is for mounting brtfs, I suppose we need to add another condition block for it instead of replacing this line.

According to https://github.com/scawp/Steam-Deck.Mount-External-Drive/pull/15#discussion_r1023667712 these flags won't work with regular ext4 drive. I tested and confirmed applying this change would fail to auto mount ex4 drives

```
mount: /run/media/deck/games: wrong fs type, bad option, bad superblock on /dev/sda1, missing codepage or helper program, or other error.
       dmesg(1) may have more information after failed mount system call.
```